src/validator.ts

```diff
@@ -1162,7 +1173,11 @@ function doIntrinsicImportValue(ref: any, key: string){
 
     // Resolve
     if(typeof toGet == 'string'){
-        return "IMPORTEDVALUE" + toGet; // TODO: Consider making this commandline defined
+        const importValue = importRuntimeOverride[toGet];
+        if(importValue !== undefined) {
+            return importValue;
+        }
+        return "IMPORTEDVALUE" + toGet;
```

Comment:
Can you add a comment like:
`\\ Backward compatibility, do not raise an error if the import value is not provided.`